dotnet-roslyn/config/Audacia.CodeAnalysis/Base/.editorconfig

```diff
@@ -2233,4 +2233,283 @@ dotnet_diagnostic.IDE0051.severity = none
+
+# Use compound assignment
+dotnet_diagnostic.IDE0054.severity = warning
+
+# Fix formatting
+dotnet_diagnostic.IDE0055.severity = warning
+
+# Use index operator
+dotnet_diagnostic.IDE0056.severity = warning
+
+# Use range operator
+dotnet_diagnostic.IDE0057.severity = warning
+
+# Remove unused expression value
+dotnet_diagnostic.IDE0058.severity = warning
+
+# Remove unnecessary value assignment
+dotnet_diagnostic.IDE0059.severity = warning
+
+# Remove unused parameter
+dotnet_diagnostic.IDE0060.severity = warning
+
+# Use expression body for local functions
+dotnet_diagnostic.IDE0061.severity = warning
+
+# Make local function static
+dotnet_diagnostic.IDE0062.severity = warning
+
+# Use simple using statement
+dotnet_diagnostic.IDE0063.severity = warning
+
+# Make struct fields writable
+dotnet_diagnostic.IDE0064.severity = warning
+
+# using directive placement
+dotnet_diagnostic.IDE0065.severity = warning
+
+# Use switch expression
+dotnet_diagnostic.IDE0066.severity = warning
+
+# Use System.HashCode.Combine
+dotnet_diagnostic.IDE0070.severity = warning
+
+# Simplify interpolation
+dotnet_diagnostic.IDE0071.severity = warning
+
+# Add missing cases to switch expression
+dotnet_diagnostic.IDE0072.severity = warning
+
+# Use file header
+dotnet_diagnostic.IDE0073.severity = warning
+
+# Use coalesce compound assignment
+dotnet_diagnostic.IDE0074.severity = warning
+
+# Simplify conditional expression
+dotnet_diagnostic.IDE0075.severity = warning
+
+# Remove invalid global SuppressMessageAttribute
+dotnet_diagnostic.IDE0076.severity = warning
+
+# Avoid legacy format target in global SuppressMessageAttribute
+dotnet_diagnostic.IDE0077.severity = warning
+
+# Use pattern matching
+dotnet_diagnostic.IDE0078.severity = warning
+
+# Remove unnecessary suppression
+dotnet_diagnostic.IDE0079.severity = warning
+
+# Remove unnecessary suppression operator
+dotnet_diagnostic.IDE0080.severity = warning
+
+# Remove ByVal
+dotnet_diagnostic.IDE0081.severity = warning
+
+# Convert typeof to nameof
+dotnet_diagnostic.IDE0082.severity = warning
+
+# Use pattern matching (not operator)
+dotnet_diagnostic.IDE0083.severity = warning
+
+# Use pattern matching (IsNot operator)
+dotnet_diagnostic.IDE0084.severity = warning
+
+# Simplify new expression
+dotnet_diagnostic.IDE0090.severity = warning
+
+# Remove unnecessary equality operator
+dotnet_diagnostic.IDE0100.severity = warning
+
+# Remove unnecessary discard
+dotnet_diagnostic.IDE0110.severity = warning
+
+# Simplify LINQ expression
+dotnet_diagnostic.IDE0120.severity = warning
+
+# Namespace does not match folder structure
+dotnet_diagnostic.IDE0130.severity = warning
+
+# Simplify object creation
+dotnet_diagnostic.IDE0140.severity = warning
+
+# Prefer null check over type check
+dotnet_diagnostic.IDE0150.severity = warning
+
+# Use block-scoped namespace
+dotnet_diagnostic.IDE0160.severity = warning
+
+# Use file-scoped namespace
+dotnet_diagnostic.IDE0161.severity = warning
```

Comment:
Similarly `IDE0161` and `IDE0160` contradict - we either want block-scoped or file-scoped namespaces.

Pretty sure file-scoped is our preference since it was added so `IDE0160` should be set to `none`?

Reply:
Yep great spot - updated.